### ISSUE TITLE
Tests in CI never finishes to run

### ISSUE THREAD

**Issue by VictorBjelkholm** [ipfs-shipyard/ipld-explorer#4]
The CI job runs the Makefile and the Makefile specifies to run `npm run test`, but since it's a interactive command and not meant to be run unattended, it hangs on the following part:

```
Watch Usage
 › Press a to run all tests.
 › Press p to filter by a filename regex pattern.
 › Press t to filter by a test name regex pattern.
 › Press q to quit watch mode.
 › Press Enter to trigger a test run.
Sending interrupt signal to process
```

**Comment by olizilla** [comment]
Ah, travis and circle both set an ENV var of CI=true

- https://docs.travis-ci.com/user/environment-variables/#default-environment-variables
- https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables

@VictorBjelkholm can we set CI=true on jenkins too pls?

**Comment by VictorBjelkholm** [comment]
Yup, can do! We already do for [JS jobs](https://github.com/ipfs/jenkins-libs/blob/master/vars/javascript.groovy#L20) but not for website jobs it seems.

**Comment by olizilla** [comment]
Fixed in #5
